src/plugins/shared_ux/public/components/empty_state/no_data_views/no_data_views_component.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { lazy, Suspense } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButton, EuiEmptyPrompt, EuiLoadingSpinner, EuiEmptyPromptProps } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { DocumentationLink } from './documentation_link';
+
+const Illustration = lazy(() => import('../assets/data_view_illustration'));
+
+interface Props {
+  canCreateNewDataView: boolean;
+  onClickCreate?: () => void;
+  dataViewsDocLink?: string;
+  emptyPromptColor?: EuiEmptyPromptProps['color'];
+}
+
+const createDataViewText = i18n.translate('sharedUX.noDataViewsPage.addDataViewText', {
+  defaultMessage: 'Create Data View',
+});
+
+export const NoDataViewsComponent = ({
+  onClickCreate,
+  canCreateNewDataView,
+  dataViewsDocLink,
+  emptyPromptColor = 'plain',
+}: Props) => {
+  // Using raw value because it is content dependent
+  const maxWidth = '830px';
+
+  const createNewButton = canCreateNewDataView && (
+    <EuiButton
+      onClick={onClickCreate}
+      iconType="plusInCircle"
+      fill={true}
+      data-test-subj="createDataViewButtonFlyout"
+    >
+      {createDataViewText}
+    </EuiButton>
+  );
+
+  return (
+    <EuiEmptyPrompt
+      data-test-subj="noDataViewsPrompt"
+      layout="horizontal"
+      css={css`
+        max-width: ${maxWidth} !important;
```

Comment:
...and this would become:

```suggestion
        max-width: ${MAX_WIDTH}px !important;
```

@cchaos what's our stance on `!important`...?

Reply:
Sometimes it's necessary to override specificity. In Sass, we have a linter that by default doesn't allow them, but you can comment the linter to ignore by also adding a reason. So something like 

```suggestion
        max-width: ${MAX_WIDTH}px !important; // Necessary to override EuiEmptyPrompt to fit content
```